src/components/Table/Table.vue

```diff
@@ -4,46 +4,50 @@
   </table>
 </template>
 
-<script>
-export default {
-  name: 'LobTable',
-  props: {
-    space: {
-      type: String,
-      default: 'md',
-      validator: function (value) {
-        return ['sm', 'md', 'lg'].includes(value);
-      }
-    }
-  },
-  computed: {
-    spaceClassName() {
-      return `space-${this.space}`;
-    }
-  },
-  methods: {}
-};
+<script setup lang="ts">
+import { computed, defineOptions } from 'vue';
+import { TableSize } from './constants';
+
+defineOptions({ name: 'LobTable' });
+
+const props = withDefaults(
+  defineProps<{
+    /** @deprecated Utilize the `size` prop. */
+    space?: TableSize;
```

Comment:
probs worth fixing this before merging, even if we just set it to null

Reply:
Whoops! I missed that, will fix now 😄